app/models/solr_document.rb

```diff
@@ -28,4 +26,12 @@ class SolrDocument
 
   # Do content negotiation for AF models.
   use_extension(Hydra::ContentNegotiation)
+
+  # overrides the MimeTypes included by CurationConcerns::SolrDocumentBehavior in order to add psd types.
+
+  module Hydra::Works::MimeTypes::ClassMethods
```

Comment:
If you're changing a behavior of an included module, you should override the method or methods that are defined in the module. I'm also not clear on why this is needed. I'm not seeing a corresponding test to solr_document. Could you write a test that demonstrates a failure if this particular bit of code is not present?